production/ksonnet/loki/ingester.libsonnet

```diff
@@ -47,42 +46,19 @@ local k = import 'ksonnet-util/kausal.libsonnet';
     container.mixin.readinessProbe.httpGet.withPort($._config.http_listen_port) +
     container.mixin.readinessProbe.withInitialDelaySeconds(15) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
-    k.util.resourcesRequests('1', '5Gi') +
-    k.util.resourcesLimits('2', '10Gi') +
+    k.util.resourcesRequests('1', '7Gi') +
```

Comment:
I wonder if we should make this configurable.

Reply:
Same here. These are just default values